src/components/Article/index.tsx

```diff
@@ -18,6 +18,7 @@ interface Props {
   previous?: PaginationInfo;
   blog?: boolean;
   date?: string;
+  children?: JSX.Element;
```

Comment:
I think the correct type for children should be `React.ReactNode`

Reply:
thanks, fixed